### PR TITLE
Add translate key for Forge pack.mcmeta description (feature of 1.19.3)

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
This PR add a translation key for the description of Forge pack.mcmeta (new feature of 1.19.3).

This makes it possible to be in agreement with the other data packs which also have a translation key.

### CODE REVIEW

**Comment by Mysterious-Dev** [comment]
I didn't dare to integrate a data generator for the pack.mcmeta (like Minecraft does now). Tell me if it's better to do it or not.

File: src/main/resources/assets/forge/lang/en_us.json [hunk 1]
```diff
@@ -218,5 +218,7 @@
   "fluid_type.minecraft.flowing_milk": "Milk",
 
   "forge.experimentalsettings.tooltip": "This world uses experimental settings, which could stop working at any time.",
-  "forge.selectWorld.backupWarning.experimental.additional" : "This message will not show again for this world."
+  "forge.selectWorld.backupWarning.experimental.additional" : "This message will not show again for this world.",
+
+  "pack.forge.description": "Forge resources pack"
```
Comment (ChampionAsh5357) [inline src/main/resources/assets/forge/lang/en_us.json]:
```suggestion
  "pack.forge.description": "Forge resource pack"
```

**Review by ChampionAsh5357 — CHANGES_REQUESTED** [review]
Vanilla already has a data provider for pack metadata which Forge hooks into, so I don't think it's necessary.

File: src/main/resources/assets/forge/lang/en_us.json (hunk 1 above)
Comment (Mysterious-Dev) [reply src/main/resources/assets/forge/lang/en_us.json]: It's modified.

**Review by ChampionAsh5357 — APPROVED** [review]
@TheCurle if you pull this, can you make sure to add the key on crowdin? Thanks.

**Pull request merged**